### PR TITLE
updated eclipse license in gradle and bundle files

### PR DESCRIPTION
**Pull request by Wyvinar**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-extensions-pr-205-bdb4s for details. If you are unable to do so, please contact a member of the Galasa team.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```
Comment (techcobweb) [inline .vscode/settings.json:2]: This is odd, we don't generally check these things into github...

Comment (techcobweb) [reply .vscode/settings.json:2]: Can you remove the .json folder, and add it to .gitignore please ?

The idea is that people can use any IDE they want, so having vscode-specific stuff in there isn't necessary I think.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-extensions-pr-205-bdb4s-r-8746r for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-extensions-pr-205-n27qz for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]

**Pull request merged**